safe/impact_function/multi_exposure_wrapper.py

```diff
@@ -282,8 +296,8 @@ def aggregation_summary(self):
         return self._aggregation_summary
 
     @property
-    def analysis_summary(self):
-        """Property for the aggregation summary.
+    def analysis_impacted(self):
```

Comment:
We try to move to the new names: `analysis_summary` https://github.com/inasafe/inasafe/blob/develop/safe/definitions/layer_purposes.py#L134
Why did you change this one?

Reply:
I just try to make it matches the `ImpactFunction` , IMHO we should have the same property name for the same purpose, either to change what's in `ImpactFunction` or change what's in `MultiExposureImpactFunction` . no pressure, very flexible :smile:

Reply:
Agree. `analysis_impacted` was the old name, before the release of 4.0. Better to update `ImpactFunction` I think

Reply:
can I do this in another PR?

Reply:
So why not revert back this one?

Reply:
Because you need to use the same property in `ImpactReport` ?

Reply:
I use this somewhere in the code actually, ok let me revert this back and make some conditional code first

Reply:
> Because you need to use the same property in ImpactReport ? Go for it, it's fine

yes,  you know me so well :smiley:  ok then, it's easier to refactor if we leave it like this